app/scripts/services/roles.factory.js

```
@@ -33,36 +29,48 @@
+  roles.notConfiguredUsers = [];
+  roles.notConfiguredUsersMessage = '';
 
-  roles.current = {
-    role: localStorage.getItem('role') || setDefaultRole(),
-    roles: roles.ROLES[(localStorage.getItem('role') || 1) - 1],
+  const updateMessage = () => {
+    if (!roles.notConfiguredUsers.length) {
+      roles.notConfiguredUsersMessage = '';
+    } else {
+      roles.notConfiguredUsersMessage =
+        'app.errors.not-configured-' + roles.notConfiguredUsers.join('-');
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)